conversation_reconstruction_dataflow/conversation_reconstruction/construct_utils.egg-info/SOURCES.txt

```diff
@@ -1,7 +1,8 @@
-README.rst
+README.md
```

Comment:
Remind me what the '.egg-info' postfix for this subdirectory is for (the directory that this file is in)?  Want to make sure this isn't an accidentally checked in directory and file... same with next file :)

Reply:
I think this directory was automatically generated while uploading the pipeline as a python package to the dataflow.

Reply:
If it's generated, then I don't imagine you actually want it in github then? remove it?

Reply:
Yep, I removed them and put them into gitignore. will commit later.